app/views/data_exports/show.html.erb

```diff
@@ -0,0 +1,101 @@
+<%= render Viral::PageHeaderComponent.new(
+  title: @data_export.name || @data_export.id, id: @data_export.name.nil? ? nil : @data_export.id) do |component| %>
+  <%= component.with_buttons do %>
+    <div class="flex flex-row">
+      <% if @data_export.status == 'ready' %>
+        <%= link_to(
+          t("data_exports.show.download"),
+          download_data_export_path(@data_export),
+          class: "button button--state-primary button--size-default"
+        ) %>
+      <% else %>
+        <%= link_to(
+          t("data_exports.show.download"),
+          download_data_export_path(@data_export),
+          class:"
+            button
+            button--size-default
+            button--state-default
+            pointer-events-none
+            cursor-not-allowed
+            bg-slate-100
+            text-slate-600
+            dark:bg-slate-600
+            dark:text-slate-300
+            border-slate-100
+            dark:border-slate-600
+          "
+        ) %>
+      <% end %>
+        <%= link_to(
+          t("data_exports.show.remove_button"),
+          remove_data_export_path(@data_export),
+          data: {
+            turbo_method: :delete,
+            turbo_confirm: t("data_exports.show.remove_button_confirmation", name: @data_export.name || @data_export.id)
+          },
+          class: "button button--state-default button--size-default mr-1 ml-2"
+        ) %>
+    </div>
+  <% end %>
+<% end %>
+
+<div class="bg-white dark:bg-slate-800">
+  <div
+    class="
+      flex
+      text-sm
+      font-medium
+      text-center
+      border-b
+      text-slate-500
+      border-slate-200
+      dark:text-slate-400
+      dark:border-slate-700
+    "
+  >
+    <ul class="flex flex-wrap -mb-px">
+      <li class="mr-2">
+        <%= link_to t(:"data_exports.show.tabs.attributes"),
+        data_export_path(@data_export, tab: "attributes"),
+        class:
+          (
+            if @tab == "attributes" || @tab == nil
+              "inline-block p-4 text-primary-700 border-b-2 border-primary-700 rounded-t-lg active dark:text-primary-500 dark:border-primary-500"
+            else
+              "inline-block p-4 border-b-2 border-transparent rounded-t-lg hover:text-slate-600 hover:border-slate-300 dark:hover:text-slate-300"
+            end
+          ),
+        aria: {
+          current: (@tab == "attributes" || @tab == nil) ? "page" : "false"
+        } %>
+      </li>
+      <li class="mr-2">
+        <%= link_to t(:"data_exports.show.tabs.manifest"),
+        data_export_path(@data_export, tab: "manifest"),
+        class:
+          (
+            if @tab == "manifest" && @data_export.status == 'ready'
+              "inline-block p-4 text-primary-700 border-b-2 border-primary-700 rounded-t-lg active dark:text-primary-500 dark:border-primary-500"
+            elsif @data_export.status == 'processing'
```

Comment:
**suggestion:** Let's just hide this tab if the Manifest is not available.

Reply:
Fixed in e6e2a039de410dc1535686ca33df7ac42470e1eb